tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -276,29 +270,21 @@ private HttpRequestMessage CreateUpstreamRequest(HttpRequest incomingRequest, by
             {
                 IEnumerable<string> values = header.Value;
 
-                if (s_excludedRequestHeaders.Contains(header.Key, StringComparer.OrdinalIgnoreCase))
```

Comment:
Why was this removed?

Reply:
Good point. It would handle dups.

Reply:
Actually. No. I misread this. I definitely don't want this to be here. It DOES need to handle duplicates here though. Otherwise we will tryAdd it, not (because it's already there), and go to add it to content erroneously.

Reply:
Not sure I follow - this list contains headers that we don't want to include on the request, so why are we removing the filter? If we should remove it, should we also delete the field?